src/atoms/Button/buttons.module.scss

```diff
@@ -30,36 +30,25 @@ $brand-white: color('primary-2');
   width: 100%;
   outline: none;
   white-space: normal;
-  border-radius: 0;
 
   &:active,
   &:focus {
     outline: none;
   }
 
-  .icon {
-    path {
-      transition: fill 300ms ease-out;
-      fill: $brand-white;
-    }
+  .icon path {
+    transition: fill 300ms ease-out;
+    fill: $brand-white;
   }
 
   &.outline {
     background-color: transparent;
     color: $action-color;
     border: 2px solid inherit;
-  }
-}
 
-.disabled {
```

Comment:
I moved the `.disabled` styling down to the bottom, so it'd apply to any variant. Previously, an outlined + `variant="info"` button wouldn't appear disabled

Reply:
bless 🙏